lib.js

```diff
@@ -25,6 +25,62 @@ function withOptions (options) {
   var db = new PouchDB(options.database)
   var configurationId = 'sense-dispatch-configuration'
   var client = new raven.Client(sentryEndpoint)
+  // change feeds timeout, and Pouch will not store the last sequence
+  // number for us. The reactive logic will restart the change feed,
+  // but in that short time interval we might lose some events if we
+  // just used "now" as a parameter for `since`. Thus we need to keep
+  // track of the last sequence number for every change we follow. See
+  // also #12
+  function followerFactory (name, options) {
+    var since = 'now'
+    // sometimes we might not receive any `change` event, thus in
+    // order to get a recent value for `since` in any case, we create
+    // a changes feed which has `live` not set, and we intercept the
+    // `complete` event
+    var oneShotOptions = _.defaults(options, { since: 'now' })
+    db.changes(oneShotOptions).on('complete', function (info) {
+      since = info.last_seq
+      log.debug(name + ' feed stored sequence number ' + since)
+    })
+    return function () {
+      // some options are always used
+      var complete = _.defaults(options, {
+        live: true,
+        include_docs: true,
+        since: since
```

Comment:
Does this rely on the oneshot changes to have completed and set the `since` value or do you anticipate that the first time `since=now` and afterwards, if the changes event `complete` above is fired, the since will be set for the next construction of the follower? I do not really understand the logic, since the oneshot is run async and sets `since`, but the control-flow does not wait for the event handler and just continues. And `since` is then used in the _second_ changes follower.